src/_C011.ino

```diff
@@ -1,6 +1,6 @@
 #ifdef USES_C011
 //#######################################################################################################
-//########################### Controller Plugin 011: Generic HTTP #######################################
+//########################### Controller Plugin 011: Generic HTTP Advanced ##############################
```

Comment:
Advanced?

Reply:
That's how the controller is being listed. I just updated the comment since I've fallen in that trap way too often

Reply:
I see, me 2 ;)